crates/ide/src/hover/tests.rs

```diff
@@ -3399,6 +3399,7 @@ impl<const LEN: usize> Foo<LEN$0> {}
     );
 }
 
+// FIXME: move these tests to consteval module
```

Comment:
yeah, we should have tests for this in consteval. probably something like the inference tests where you just have comments like `// ^^^ 27`